### DISCUSSION TITLE
Welcome to WhatsApp-Clone-Flutter Discussions!

### DISCUSSION THREAD

**Discussion by AbhilashTUofficial** [AbhilashTUofficial/WhatsApp-Clone-Flutter#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sacksaini** [comment]
I'm a newbie in Flutter, Just wanna know if we can make Hybrid apps using it? or is it just for Native apps and can't be manipulated into the Web?

**Reply by AbhilashTUofficial** [reply]
You can make cross plate form apps with it, IOS,  Android, and also web.
Flutter is a Hybrid programming language, So you just have to enable it on the editor then you can make cross platform apps.

**Reply by AbhilashTUofficial** [reply]
Welcome to flutter!!!
